app/views/provider_interface/decisions/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :browser_title, title_with_error_prefix(t('.title'), @wizard.errors.any?) %>
+<% content_for :before_content, govuk_back_link_to(provider_interface_application_choice_path(@application_choice)) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">
+      <span class="govuk-caption-l"><%= @application_choice.application_form.full_name %></span>
+      <%= t('.title') %>
+    </h1>
+    <%= render ProviderInterface::CourseSummaryComponent.new(course_option: @application_choice.course_option) %>
+
+    <%= form_with model: @wizard, url: provider_interface_application_choice_decision_path(@application_choice), method: :post do |f| %>
+      <%= f.govuk_error_summary %>
+
+      <%= f.govuk_radio_buttons_fieldset :decision, legend: { text: t('.select'), size: 'm' } do %>
+        <%= f.govuk_radio_button :decision,
```

Comment:
Do we need something like this here as well?
```
if ApplicationStateChange.new(@application_choice).can_make_offer?
```

Reply:
I don't think we do as there are pre-existing checks in the application_choice_header_component, ensuring that the Make Decision button is only displayed if the application_choice is in a decision making state (either [awaiting_provider_decision](https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/components/provider_interface/application_choice_header_component.html.erb#L34) or [interviewing](https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/components/provider_interface/application_choice_header_component.html.erb#L10). 
 
As part of this PR we are also enforcing this in the DecisionsController.

Based on this, is seems like the condition that we moved across from the old Make offer flow, around displaying the Rejection radio button, is the one not required, so I will remove it.